spring-batch-core/src/main/java/org/springframework/batch/core/launch/support/SimpleJobOperator.java

```diff
@@ -425,7 +425,7 @@ public boolean stop(long executionId) throws NoSuchJobExecutionException, JobExe
 			}
 		}
 		catch (NoSuchJobException e) {
-			logger.warn("Cannot find Job object",e);
+			logger.warn("Cannot find Job object in the job registry",e);
```

Comment:
We can probably be even more detailed here.  Something like `"Cannot find Job object in the job registry.  StoppableTasklet#stop() will not be called"`.

Reply:
Sure! I will update the PR accordingly.

Reply:
PR updated. Thank you for the review!